testproject/Assets/Tests/Runtime/MultiprocessRuntime/Helpers/MultiprocessOrchestration.cs

```diff
@@ -43,10 +66,13 @@ public static void StartWorkerNode()
             throw;
         }
 
+        string logPath = Path.Combine(s_MultiprocessDirInfo.FullName, $"zlogfile{Processes.Count}");
```

Comment:
intentional `(z)logfile` prefix in the name?

Reply:
I just wanted to make it unique enough to be easily seen. I can call it logfile or whatever seems best.

Reply:
oh, I see. perhaps `$"logfile-mp{Processes.Count}"`?

Reply:
done